### PR TITLE
Add recipes for Perl Moose deps, part 2

### PR DESCRIPTION
**Pull request by xileF1337**
This is the second of a row of PRs providing recipes for Perl module Moose and its dependencies.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xileF1337** [comment]
@conda-forge/bioconda-recipes  I'm porting these recipes to CondaForge unless there are any objections from your side.

**Comment by xileF1337** [comment]
@cbrueffer The second batch is incoming -- would you be so kind as to review these recipes as well? Thank you ;-)

File: recipes/perl-sub-name/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+set -o errexit -o pipefail
```
Comment (cbrueffer) [inline recipes/perl-sub-name/build.sh:2]: This one misses the shebang line and has end-of-file whitespace, but let's deal with that after merging.

**Comment by cbrueffer** [comment]
Sorry for the delay, the end of the year rush is keeping me busy ;-)

**Comment by xileF1337** [comment]
Awesome @cbrueffer, thanks for your work! But has the CI done anything yet? It's been 20h since merging and it seems the repos for the feedstocks (e.g. https://github.com/conda-forge/perl-module-build-feedstock) have been created, but they are all empty? :shrug:

**Comment by xileF1337** [comment]
The CI seems to be back up. Feedstocks are filled now. :+1:

**Pull request merged**